app/javascript/hyacinth_ui_v1/graphql/file_uploads.js

```diff
@@ -0,0 +1,20 @@
+import * as ActiveStorage from 'activestorage';
```

Comment:
If a file has a single default export of a variable, I tend to give the file the same name as the single default export, so I'd recommend renaming to `uploadFile.js`.

Reply:
I think there will be other functions here when I decompose after Rails 6

Reply:
👍